RoslynPluginGenerator/NuGet/NuGetRepositoryFactory.cs

```diff
@@ -60,7 +66,8 @@ public static IPackageRepository CreateRepository(ISettings settings, Common.ILo
                 }
             }
             // Create an aggregate repository that uses all of the configured sources
-            AggregateRepository aggRepo = packageSourceProvider.CreateAggregateRepository(PackageRepositoryFactory.Default, true /* ignore failing repos */ );
+            AggregateRepository aggRepo = packageSourceProvider.CreateAggregateRepository(PackageRepositoryFactory.Default, false /* don't ignore failing repos */ );
```

Comment:
For the SDK, we won't ignore failing repos i.e. exceptions will not be suppressed. This makes it easier to diagnose e.g. authentication issues (if _ignore_ is _true_ then there is no log output at all in the case of authentication errors which makes troubleshooting very difficult).